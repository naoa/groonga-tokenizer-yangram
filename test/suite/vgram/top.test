register tokenizers/yangram

table_create vgram_words TABLE_HASH_KEY ShortText
load --table vgram_words
[
{"_key": "画像"}
]
tokenize TokenYaVgram "画像" NormalizerAuto --mode ADD
tokenize TokenYaVgram "画像" NormalizerAuto --mode GET
tokenize TokenYaVgram "画像処" NormalizerAuto --mode ADD
tokenize TokenYaVgram "画像処" NormalizerAuto --mode GET
tokenize TokenYaVgram "画像処理" NormalizerAuto --mode ADD
tokenize TokenYaVgram "画像処理" NormalizerAuto --mode GET
