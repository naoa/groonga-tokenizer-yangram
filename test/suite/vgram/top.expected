register tokenizers/yangram
[[0,0.0,0.0],true]
table_create #vgram_words TABLE_HASH_KEY ShortText
[[0,0.0,0.0],true]
load --table #vgram_words
[
{"_key": "画像"}
]
[[0,0.0,0.0],1]
tokenize TokenYaVgram "画像" NormalizerAuto --mode ADD
[[0,0.0,0.0],[{"value":"画像","position":0},{"value":"像","position":1}]]
tokenize TokenYaVgram "画像" NormalizerAuto --mode GET
[[0,0.0,0.0],[{"value":"画像","position":0}]]
tokenize TokenYaVgram "画像処" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "画像",
      "position": 0
    },
    {
      "value": "画像処",
      "position": 1
    },
    {
      "value": "像処",
      "position": 2
    },
    {
      "value": "処",
      "position": 3
    }
  ]
]
tokenize TokenYaVgram "画像処" NormalizerAuto --mode GET
[[0,0.0,0.0],[{"value":"画像処","position":1}]]
tokenize TokenYaVgram "画像処理" NormalizerAuto --mode ADD
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "画像",
      "position": 0
    },
    {
      "value": "画像処",
      "position": 1
    },
    {
      "value": "像処",
      "position": 2
    },
    {
      "value": "処理",
      "position": 3
    },
    {
      "value": "理",
      "position": 4
    }
  ]
]
tokenize TokenYaVgram "画像処理" NormalizerAuto --mode GET
[
  [
    0,
    0.0,
    0.0
  ],
  [
    {
      "value": "画像処",
      "position": 1
    },
    {
      "value": "処理",
      "position": 3
    }
  ]
]
